static/assets/icons/zulip-icons.font.js

```diff
@@ -3,7 +3,10 @@
 "use strict";
 
 module.exports = {
-    files: ["./*.svg"],
+    files: [
+        "./*.svg", // For web-only icons.
```

Comment:
Why not move this whole directory? I don't think we have a need to have special "web-only" icons, and it seems simpler that way.

Reply:
(And one could imagine the mobile apps wanting to build the icon font).

Reply:
I agree we probably don't need to have any web-only icons.

We do plan to build an icon font in mobile once these are shared, but it's not likely to be the same icon font or use the same build instructions (this file and the other remaining file in this directory, `template.hbs`, which is a template for CSS rules.) See the chat thread linked above, or starting from this comment by Anders earlier in that thread:
https://chat.zulip.org/#narrow/stream/243-mobile-team/topic/custom.20icons/near/1338098

Because these two files don't really make a lot of sense on their own without the context of the web build config that invokes them, it'd feel a bit messy to have them in the shared package. But I wouldn't really mind that messiness very much in practice, if it feels significantly nicer/easier from the web side.